dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -460,16 +462,16 @@ public boolean isUnmodifiableByOtherCode() {
     }
 
     public static class LocalVariable extends Receiver {
-        protected Element element;
+        protected VarSymbol element;
```

Comment:
I would keep the field an Element and downcast only in the very limited places where needed.